src/commands/generate/types.ts

```diff
@@ -230,6 +239,34 @@ export const ${constantName} = '${hashedKey}' as const`
     private encryptKey(variable: Variable) {
         return `dvc_obfs_${createHash('sha256').update(variable._id).digest('hex')}`
     }
+
+    private updateFileLocation() {
+        if (!this.repoConfig) {
+            return
+        }
+        const fileLocation = this.repoConfig.typeGenerator?.fileLocation
+        const newFileLocation = path.join(
+            this.outputDir,
+            '/dvcVariableTypes.ts',
+        )
+        if (fileLocation !== newFileLocation) {
+            this.updateRepoConfig({
+                typeGenerator: {
+                    ...this.repoConfig.typeGenerator,
+                    fileLocation: newFileLocation,
+                },
+            })
+            if (fileLocation) {
+                this.writer.successMessage(
+                    `Updated configured types file location to ${newFileLocation}`,
+                )
+            } else {
+                this.writer.successMessage(
+                    `Stored configured types file location as ${newFileLocation}`,
+                )
+            }
```

Comment:
Just a nitpick, but it seemed unnecessarily verbose
```suggestion
            const message = fileLocation
                ? `Updated configured types file location to ${newFileLocation}`
                : `Stored configured types file location as ${newFileLocation}`
            this.writer.successMessage(message)
```